index.html

```diff
@@ -7,6 +7,7 @@
     <link rel="icon" type="image/png" href="./playcanvas-logo.png" />
     <link rel="stylesheet" href="https://code.playcanvas.com/pcui.css">
     <link rel="stylesheet" href="style.css">
+    <script src="./playcanvas.js"></script>
```

Comment:
Maybe just make this `https://code.playcanvas.com/playcanvas-stable.js` until @ellt92 switches the viewer to grab the engine from NPM?

Reply:
Could do, but viewer may not work until next engine release (due to meshopt changes).